packages/effection/src/index.ts

```diff
@@ -3,3 +3,4 @@ export { Mailbox } from './mailbox';
 export { any } from './pattern';
 export { suspend } from './suspend';
 export { ensure } from './ensure';
+export { main } from './node';
```

Comment:
Should this be exported from `index`? Would this cause the agent to bundle a bunch of node-specific code we don't want?

Reply:
Ugh. You're correct... JavaScript bundling is the worst.